lib/adapters/serializers/csv.js

```diff
@@ -0,0 +1,16 @@
+var _ = require('underscore');
+var base = require('./base');
+
+module.exports = base.extend({
+
+    write: function(points) {
+        var self = this;
+        _.each(points, function(point) {
+            var line = [];
+            _.each(point, function(value, key) {
```

Comment:
The csv serializer needs to write a header line with the field names, and then write out the fields in the right order (iterating over the fields of an object isn't guaranteed to always return them in the same order).